pdp/static/js/pdp_auth.js

```diff
@@ -185,79 +183,133 @@ window.pdp = (function (my, $) {
         pdp.createCookie(name, "", -1);
     };
 
-    my.getLoginForm = function (providers) {
+    my.validateEmail = function (email) {
+        var re = /^([\w-]+(?:\.[\w-]+)*)@((?:[\w-]+\.)*\w[\w-]{0,66})\.([a-z]{2,6}(?:\.[a-z]{2})?)$/i;
```

Comment:
Since you're not capturing any groups can you remove group parens that aren't modified by number parameters? Make it a little easier to read? For example in the user section this:
```([\w-]+(?:\.[\w-]+)*)```
can be replaced with this:
```[\w-]+(?:\.[\w-]+)*``` which is the same if you're not capturing groups.